main.go

```diff
@@ -192,16 +194,52 @@ func readSnippets(config Config) []Snippet {
 	return snippets
 }
 
+// migrateSnippets migrates any legacy snippet <dir>-<file> format to the new <dir>/<file> format
+func migrateSnippets(config Config, snippets []Snippet) []Snippet {
+	var migrated bool
+	for idx, snippet := range snippets {
+		legacyPath := filepath.Join(config.Home, snippet.LegacyPath())
+		if _, err := os.Stat(legacyPath); err != nil {
+			if !errors.Is(err, fs.ErrNotExist) {
+				fmt.Printf("could not access %q: %v\n", legacyPath, err)
+			}
+			continue
+		}
+		file := strings.TrimPrefix(snippet.LegacyPath(), fmt.Sprintf("%s-", snippet.Folder))
+		newDir := filepath.Join(config.Home, snippet.Folder)
+		newPath := filepath.Join(newDir, file)
+		if err := os.Mkdir(newDir, os.ModePerm); err != nil {
+			fmt.Printf("could not create %q: %v\n", newDir, err)
```

Comment:
This will print if the folder exists, so it is a bit noisy. I think we can silently ignore this.

Reply:
Replaced with `MkdirAll`, which shouldn't error if the dir already exists, but will still print if there is a problem creating the dir otherwise.

Is that better, or would you rather ignore the error here?

Reply:
I think that's the better solution!